packages/transcript-view/src/duration-formatter.ts

```diff
@@ -0,0 +1,27 @@
+import { LitElement, html, customElement, property, TemplateResult } from 'lit-element';
+
+@customElement('duration-formatter')
+export default class DurationFormatter extends LitElement {
```

Comment:
question: is this something that can be reused?

Reply:
Indeed it can! I was actually considering pulling it out to its own repo, just was a bit tight on time. 😄

Reply:
I'm going to make this a follow-up task, but I'm all about pulling this out to its own component!